_includes/_inventory/resources.md

```diff
@@ -6,26 +6,23 @@ interested in isn't listed, please open an issue or contribute!
-| Org Cloud IAM Policies                  | Done          | Done          | Open          |
-| Projects                                | Done          | N/A           | Open          |
-| Project Cloud IAM Policies              | Done          | Done          | Open          |
-| Subnetworks                             | Open          | Open          | Open          |
+| [App Engine Apps](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps) | In Progress | TODO | TODO |
+| [BigQuery](https://cloud.google.com/bigquery/docs/reference/rest/v2/) | Done | In Progress | TODO |
+| [Buckets](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) | Done | Done | TODO |
+| [BucketAccessControls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) | Done | Done | TODO |
+| [Cloud SQL](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances#resource) | Done | Done | TODO |
+| [Compute Engine Backend Services](https://cloud.google.com/compute/docs/reference/latest/backendServices#resource) | Done | In Progress | TODO |
+| [Compute Engine Instances](https://cloud.google.com/compute/docs/reference/latest/instances#resource) | Done | In Progress | TODO |
+| [Compute Engine Instance Groups](https://cloud.google.com/compute/docs/reference/latest/instanceGroups#resource) | Done | In Progress | TODO |
+| [Compute Engine Instance Group Managers](https://cloud.google.com/compute/docs/reference/latest/instanceGroupManagers) | Done | In Progress | TODO |
+| [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) | Done | In Progress | TODO |
+| [Firewalls](https://cloud.google.com/compute/docs/reference/latest/firewalls) | Done | TODO | Done |
+| [Folders](https://cloud.google.com/resource-manager/reference/rest/v2beta1/folders) | Done | In Progress | TODO |
+| [GSuite Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group) | Done | Done | TODO |
+| [GSuite Group Members](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group-members) | Done | Done | TODO |
+| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | In Progress | TODO | TODO |
+| [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) | Done | In Progress | TODO |
+| [Organizations](https://cloud.google.com/resource-manager/reference/rest/v1beta1/organizations) | Done | N/A | TODO |
+| [Organizations Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) | Done | Done | TODO |
+| [Projects](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects) | Done | N/A | TODO |
+| [Projects Cloud IAM Policies](https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects/getIamPolicy) | Done | Done | TODO |
```

Comment:
Yeah, they were Open | Open | Open. I didn't see it useful to list them as there are lots of APIs we don't query.